app/src/main/java/com/connorlay/knitgrid/fragments/PatternDetailFragment.java

```diff
@@ -25,6 +25,11 @@
  * Created by connorlay on 2/26/16.
  */
 public class PatternDetailFragment extends Fragment {
+
+    public interface RowColumnListener {
```

Comment:
I'm not a big fan of this interface name. What about `CellSelectedListener` or something similar?